contents/founders/how-to-buy-enterprise-software.md

```diff
@@ -0,0 +1,225 @@
+---
+date: 2024-08-15
+title: How to buy software as an enterprise
+author:
+  - charles-cook
+  - leon-daly
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/take_my_money_11433edb48.png
+featuredImageType: full
+tags:
+  - Founders
+  - Growth
+  - Ops & finance
+crosspost:
+  - Blog
+---
+
+Buying software as an enterprise company is a pain in the ass. There's no playbook, everyone has different objectives, and the whole process is littered with surprises. If this is your first time doing this, these are some things you may not realize:
+
+- Buying software at big companies can take an insanely long time. It typically takes 6-12 months time to go from deciding you need the software to actually closing the deal.
+- Negotiating internally with your team is _much_ more important and difficult than negotiating with the salesperson. 
+- The typical enterprise software buying process involves 8-12 people - it’s not just you and a finance person who signs it off.
+- Your salesperson can do _so_ much more for you than just show you the product and negotiate pricing.
+
+Are you a software buying newbie at a large company? Have you bought software before and it went terribly and took way longer than you expected? Have you been handed something called an RFP and have no idea WTF to do with it? Do you enjoy reading enterprise software buying guides in your spare time?
+
+This guide is for you! We'll break down the entire process - everything you need to do to look good internally, but also how to handle your salesperson. As a bonus, we've also thrown in the 7 final bosses you'll (probably) need to beat at the end. 
+
+## The 5 levels you need to clear
+
+Buying enterprise software is a game, and games have levels (wait, haven't we done this [before](/founders/product-market-fit-game)?)
+
+1. I think I need software to solve a problem
+2. Shortlist the ones that could do the job
+3. Get demos & test the ones you like
+4. It’s time to negotiate
+5. How do I actually buy this thing?
+
+You need to complete all five levels to win. At each stage, we'll outline what to do internally, what to do with your salesperson, and a common trap to avoid. 
+
+Let's go!
+
+### Level 1 - I think I need software to solve a problem
```

Comment:
```suggestion
### Level 1: I think I need software to solve a problem
```

Reply:
amend here and throughout. i'd always opt for : over – as it's neater and more standard. you've used – on all the crossheads, i'd switch 'em for :